### ISSUE TITLE
Validation issues with OpenxQuest_example.mzid

### ISSUE THREAD

**Issue by edeutsch** [HUPO-PSI/mzIdentML#52]
WARNING: XL:00002 should be 'DSS' instead of 'Xlink:DSS'

ERROR: cvParam retention time has an illegal unit accession number "second"!

The latter is complaining about this:
cvParam accession="MS:1000894" cvRef="PSI-MS" name="retention time" value="5458.13539999998" unitAccession="second" unitName="" unitCvRef="se"/>

The name is written in the accession. There is no name. Also in error the unitCvRef is "se", but in the cvlist, the ID is "UO":
        <cv id="UO"     fullName="UNIT-ONTOLOGY" uri="https://raw.githubusercontent.com/bio-ontology-research-group/unit-ontology/master/unit.obo"></cv>

**Comment by edeutsch** [comment]
These same problems apply to OpenxQuest_example_added_annotations.mzid

**Comment by germa** [comment]
It seems that these examples also violate the .mzid XML schema definition

**Comment by germa** [comment]
I think for the moment it would suffice to replace

	xsi:schemaLocation="http://psidev.info/psi/pi/mzIdentML/1.2.0 https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/schema/mzIdentML1.2.0.xsd"
	xmlns="http://psidev.info/psi/pi/mzIdentML/1.2.0"

by 
	xsi:schemaLocation="http://psidev.info/psi/pi/mzIdentML/1.2 https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/schema/mzIdentML1.2.0.xsd"
	xmlns="http://psidev.info/psi/pi/mzIdentML/1.2"

**Comment by edeutsch** [comment]
and:
Error at file /net/db/projects/PSI/mzIdentML/1.2/genDoc/OpenxQuest_example.mzid, line 173, char 114
  Message: Datatype error: Type:InvalidDatatypeValueException, Message:Value 'file:///C:\MSData\S26_Yeast_RAW/aleitner_M1012_004.mzML' is NOT a valid URI .

Error at file /net/db/projects/PSI/mzIdentML/1.2/genDoc/OpenxQuest_example.mzid, line 312, char 13
  Message: The key for identity constraint of element 'MzIdentML' is not found.

**Comment by mwalzer** [comment]
Yes, unit is a problem in the way we generate that cv element that slipped my attention.

**Comment by mwalzer** [comment]
so, the version scheme of 1.2.0 to change to 1.2 is done readily, but @germa are there other schema violations my checker isnt telling me or was it that schema validation didnt find anything under 1.2.0?

**Comment by mwalzer** [comment]
one warning remains though:
```
Message 2:
    Level: WARN
    --> unanticipated terms for XPath '/MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/spectrumIdentificationResult/spectrumIdentificationItem/cvParam/@accession' : [MS:1000894]
```
which comes from the rt cv at SII instead of at SIR, which is needed, because of referencing several spectra ...

**Comment by mwalzer** [comment]
@andrewrobertjones @germa should this be reflected in the rules?

**Comment by edeutsch** [comment]
CV validation issues today:

ERROR: cvParam retention time has an illegal unit second!
(word "second" in place of accession number)

WARNING: XL:00002 should be 'cross-linker related PTM' instead of 'Xlink:DSS'
(XL accessions changed!)

**Comment by edeutsch** [comment]
also, in addition to the CV issues above, the XML name space is incorrect:

Error at file /net/db/projects/PSI/mzIdentML/1.2/genDoc/OpenxQuest_example.mzid, line 7, char 37
  Message: Schema in mzIdentML1.2.0.xsd has a different target namespace from the one specified in the instance document http://psidev.info/psi/pi/mzIdentML/1.2.0.

**Comment by mwalzer** [comment]
@edeutsch I have not merged the pull request yet, to leave the option to comment and my guess is you checked the master branch. In the PR, the issues should be resolved; [for example the seconds issue like this](https://github.com/HUPO-PSI/mzIdentML/pull/66/files#diff-c20a40dd28e60257b3216222e699b965R208) 

@edeutsch but thank you for checking, could you check the two from the branch? Here are the direct links to the files: [OpenxQuest_example.mzid](https://github.com/mwalzer/mzIdentML/raw/feature/QXQ_examples-validation_fix/examples/1_2examples/crosslinking/OpenxQuest_example.mzid) [OpenxQuest_example_added_annotations.mzid](https://raw.githubusercontent.com/mwalzer/mzIdentML/feature/QXQ_examples-validation_fix/examples/1_2examples/crosslinking/OpenxQuest_example_added_annotations.mzid)
They checked out with the validator from the repository, now I'd be curious to what your validator reads before merging.

**Comment by germa** [comment]
@mwalzer: Use of XLMOD:nnnnn instead of XL:nnnnn, see also GitHub issue #60

**Comment by andrewrobertjones** [comment]
@mwalzer  Can you fix all the issues and then close the issue. I would like to assemble all material for submission to PSI process next week.

**Comment by edeutsch** [comment]
The only remaining issue I see still is the one mentioned above. XL: needs to be XLMOD:

**Comment by mwalzer** [comment]
fixed in #77
